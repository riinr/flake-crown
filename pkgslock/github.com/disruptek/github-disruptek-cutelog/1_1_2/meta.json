{
  "name": "cutelog",
  "version": "1.1.2",
  "author": "disruptek",
  "desc": "cute logging",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 0.19.6",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.19.6"
      }
    }
  ],
  "bin": [],
  "binDir": "",
  "srcDir": "",
  "backend": "c",
  "ref": "refs/tags/1.1.2"
}