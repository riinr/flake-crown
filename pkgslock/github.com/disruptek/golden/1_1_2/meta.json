{
  "name": "golden",
  "version": "1.1.2",
  "author": "disruptek",
  "desc": "a nim benchmark",
  "license": "MIT",
  "skipDirs": [],
  "skipFiles": [],
  "skipExt": [
    "nim"
  ],
  "installDirs": [],
  "installFiles": [],
  "installExt": [],
  "requires": [
    {
      "name": "nim",
      "str": ">= 0.20.0",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.20.0"
      }
    },
    {
      "name": "cligen",
      "str": ">= 0.9.38",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.9.38"
      }
    },
    {
      "name": "foreach",
      "str": ">= 1.0.2",
      "ver": {
        "kind": "verEqLater",
        "ver": "1.0.2"
      }
    },
    {
      "name": "http://github.com/genotrance/nimgit2.git",
      "str": ">= 0.1.0",
      "ver": {
        "kind": "verEqLater",
        "ver": "0.1.0"
      }
    }
  ],
  "bin": [
    "golden"
  ],
  "binDir": "",
  "srcDir": "src",
  "backend": "c",
  "ref": "refs/tags/1.1.2"
}